cds_ils/config.py

```diff
@@ -267,63 +267,63 @@ def _parse_env_bool(var_name, default=None):
 # Namespaces for fields added to the metadata schema
 ILS_RECORDS_METADATA_NAMESPACES = {
     "document": {
-        "accelerator_experiments": {
-            "@context": "https://cern.ch/accelerator_experiments/terms"
+        "unit": {
+            "@context": "https://cern.ch/unit/terms"
         },
-        "standard_CERN_status": {
-            "@context": "https://cern.ch/standard_CERN_status/terms"
+        "standard_review": {
+            "@context": "https://cern.ch/standard_review/terms"
         },
     }
 }
 
 # Fields added to the metadata schema.
 ILS_RECORDS_METADATA_EXTENSIONS = {
     "document": {
-        "accelerator_experiments:accelerator": {
+        "unit:accelerator": {
             "elasticsearch": "keyword",
             "marshmallow": SanitizedUnicode(),
         },
-        "accelerator_experiments:curated_relation": {
+        "unit:curated_relation": {
             "elasticsearch": "boolean",
             "marshmallow": Bool(),
         },
-        "accelerator_experiments:experiment": {
+        "unit:experiment": {
             "elasticsearch": "keyword",
             "marshmallow": SanitizedUnicode(),
         },
-        "accelerator_experiments:institution": {
+        "unit:institution": {
             "elasticsearch": "keyword",
             "marshmallow": SanitizedUnicode(),
         },
-        "accelerator_experiments:legacy_name": {
+        "unit:legacy_name": {
             "elasticsearch": "keyword",
             "marshmallow": SanitizedUnicode(),
         },
-        "accelerator_experiments:project": {
+        "unit:project": {
             "elasticsearch": "keyword",
             "marshmallow": SanitizedUnicode(),
         },
-        "accelerator_experiments:study": {
+        "unit:study": {
             "elasticsearch": "keyword",
             "marshmallow": SanitizedUnicode(),
         },
-        "standard_CERN_status:CERN_applicability": {
+        "standard_review:CERN_applicability": {
```

Comment:
In this PR <https://github.com/CERNDocumentServer/cds-dojson/pull/266/files#diff-8c22f650d9678abd75eb860a77a826aaR275> there is a `standard_review:applicability`, instead of `standard_review:CERN_applicability`, and on the excel https://docs.google.com/spreadsheets/d/1OHMDiMfBD1Aam-kteIuBL9tEPK-C-Jevw-NcmGGxRvY/edit#gid=1257315908 it's also `applicability` (search for standard_status.applicability). I guess that it should change to `standard_review:applicability`. Since it's not part of the task can someone confirm it?

Reply:
I would say so (they probably forgot to mention it). To be confirmed.

Reply:
I would go for `standard_review:applicability`, since we are on CERN overlay I believe we don't have to mention it in every field.

Reply:
I completely agree, thanks!